build/zip_bundle.gni

```diff
@@ -0,0 +1,30 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import("//flutter/common/config.gni")
+
+if (flutter_runtime_mode == "jit_release") {
+  android_zip_archive_dir = "android-$target_cpu-jit-release"
+} else {
+  android_zip_archive_dir = "android-$target_cpu"
+}
+
+template("zip_bundle") {
+  action(target_name) {
```

Comment:
Please document template args in one or both ways:

* An `assert(defined(invoker.foo), "Foo must be defined")` declaration before doing anything so the required template arguments are asserted to exit.
* A comment block above the template that says what the template does and documents the required (and optional) invoker arguments.

Reply:
Done